css-fonts-4/Overview.bs

```diff
@@ -6146,7 +6146,9 @@ Web Fonts shadow Installed Fonts, so if an Installed Font has the same family na
 <h3 id="preinstalled-and-user-installed-fonts">
 Preinstalled Fonts and User-Installed Fonts</h3>
 
-Users may choose to install fonts on their devices. User-Installed Fonts are installed by an explicit action by the user, such as clicking an "Install" button or copying a file into a particular directory on their device. Such fonts are User-Installed Fonts and also are Installed Fonts. Web content authors should not expect the presence of user-installed fonts, because there is no guarantee any user will have performed the action to install a specific font. User Agents may choose to ignore User-Installed Fonts for the purpose of the <a href="#font-matching-algorithm">Font Matching Algorithm</a>.
+Users may choose to install fonts on their devices. User-Installed Fonts are installed by an explicit action by the user, such as clicking an "Install" button or copying a file into a particular directory on their device. Such fonts are User-Installed Fonts and also are Installed Fonts. Web content authors should not expect the presence of user-installed fonts, because there is no guarantee any user will have performed the action to install a specific font.
```

Comment:
s/Users may/Users can/ since this is a statement of fact not an allowance in this spec.

(I also, btw, recommend adopting https://rhodesmill.org/brandon/2012/one-sentence-per-line/ since it makes diffing a lot easier...)

Reply:
perhaps /should not expect/should not count on/